src/TwitterAds/Analytics.php

```diff
@@ -31,6 +32,32 @@ public function stats($metricGroups, $params = [])
         return $this->all_stats($this->getAccount(), [$this->getId()], $metricGroups, $params);
     }
 
+    public static function get_job_url (Account $account,  $params = [])
+    {
+
+        $count = isset($params['count']) ? $params['count'] : null ;
+        $cursor = isset($params['cursor']) ? $params['count'] : null ;
```

Comment:
On the ternary expression it should be: 

`
$cursor = isset($params['cursor']) ? $params['cursor'] : null ;
`